python/README.md

```diff
@@ -20,6 +20,11 @@ This will install ml4ir-0.0.1 (the current version) from PyPI.
 #### Docker (Recommended)
 We have set up a `docker-compose.yml` file for building and using docker containers to train models.
 
+First, change working directory to the python package
+```
+cd python/
```

Comment:
@lastmansleeping Great. Then we also just need to change line this line of the README:
`/bin/bash python/ml4ir/applications/ranking/scripts/example_run.sh`

to this (remove the `python/` directory from the command, since it will not run from the directory above):
`/bin/bash ml4ir/applications/ranking/scripts/example_run.sh`